bitnami/mongodb/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.1.4
+version: 10.1.5
```

Comment:
Please bump minor version instead since you're adding a new parameter.

Reply:
Done